DOL.WHD.Section14c.Web/src/modules/components/sectionWorkSites/sectionWorkSitesController.js

```diff
@@ -120,6 +120,7 @@ module.exports = function(ngModule) {
     this.doneAddingEmployees = function($event) {
       vm.addEmployee();
       $('.cd-panel').removeClass('is-visible');
+      $('body').removeClass('cd-panel-open'); 
```

Comment:
I'm wondering if this should call `closeSlidingPanel()` instead of modifying these classes directly.  That way all the code to successfully close the panel is in one place.